unet_attention.py

```diff
@@ -222,25 +268,38 @@ class UNet(nn.Module):
     ## U-Net
     """
 
-    def __init__(self, image_channels: int = 1, n_channels: int = 64,
-                 ch_mults = (1, 2, 2, 4),
-                 is_attn = (False,False,False,True),
-                 n_blocks: int = 2,middle_attn=True):
+    def __init__(
+        self,
+        image_channels: int = 1,
+        n_channels: int = 64,
+        ch_mults=(1, 2, 2, 4),
+        is_attn=(False, False, False, True),
+        n_blocks: int = 2,
+        middle_attn=True,
+        dropout: float = 0.05,
```

Comment:
while this is fairly obvious, do add the docstring entry since this is the "put-it-all-together" function